### PR TITLE
Add missing domains and improve wordings

### PR DESCRIPTION
**Pull request by LouiseBonnard**

### CODE REVIEW

**Comment by colinegin** [comment]
@LouiseBonnard anything to do here ?

File: ps_mbo.php [hunk 1]
```diff
@@ -650,27 +650,27 @@ public function displayModuleOptions($module, $output_type = 'link', $back = nul
...
+            $this->translationsTab['Configure'] = $this->trans('Configure', [], 'Admin.Actions');
+            $this->translationsTab['Delete'] = $this->trans('Delete', [], 'Admin.Actions');
+            $this->translationsTab['Install'] = $this->trans('Install', [], 'Admin.Actions');
+            $this->translationsTab['Uninstall'] = $this->trans('Uninstall', [], 'Admin.Actions');
+            $this->translationsTab['Would you like to delete the content related to this module ?'] = $this->trans('Are you sure you want to delete the content related to this module?', [], 'Admin.Modules.Notification');
+            $this->translationsTab['This action will permanently remove the module from the server. Are you sure you want to do this?'] = $this->trans('This action will permanently remove the module from the server. Are you sure you want to do this?', [], 'Admin.Modules.Notification');
+            $this->translationsTab['Remove from Favorites'] = $this->trans('Remove from Favorites', [], 'Admin.Actions');
+            $this->translationsTab['Mark as Favorite'] = $this->trans('Mark as Favorite', [], 'Admin.Actions');
```
[the remark this answers is not in the document]
Comment (LouiseBonnard) [reply ps_mbo.php]: @marionf, do you have a clue?

[the remark this answers is not in the document]
Comment (marionf) [reply ps_mbo.php]: Yes, buttons to put in favorites no longer exist in 1.7

[the remark this answers is not in the document]
Comment (LouiseBonnard) [reply ps_mbo.php]: @marionf, thanks for the reply, I delete those two lines in another PR!

**Comment by colinegin** [comment]
Waiting for QA

**Pull request merged**